package.json

```diff
@@ -81,7 +81,7 @@
         "@storybook/storybook-deployer": "^2.8.6",
         "babel-loader": "^8.0.6",
         "css-loader": "^2.1.1",
-        "formik": "^1.5.8",
+        "formik": "^2.2.5",
```

Comment:
Any reason both are not in `2.2.6` ?

Reply:
Ah, I ran `npm install --save "formik@^2.2.5"`, since that it what Sonora is using, and this is what npm updated things to, since `2.2.6` is compatible with `2.2.5`.

I can bump both to `2.2.6` though.